00-get-started/02-ingest-pg-cdc.md

```diff
@@ -12,6 +12,22 @@ Once you have the PostgreSQL server installed, connect to a database. By default
 psql -h localhost -p 5432 -d postgres -U postgres
 ```
 
+> NOTE: Specifically for Ubuntu users, if you are prompted for a password when running the `psql` command above, but you have not yet specified any password after installing PostgreSQL, check in the `pg_hba.conf` file (normally located as `/etc/postgresql/16/main/pg_hba.conf`) whether the default authentication method for the IPv4 local connections is `trust`:
```

Comment:
Mention an issue that specifically Ubuntu users need to pay attention to. Refer to [here](https://stackoverflow.com/a/26735105) for more context.